telegram/user.py

```diff
@@ -99,3 +99,22 @@ def get_profile_photos(self, *args, **kwargs):
         Shortcut for ``bot.getUserProfilePhotos(update.message.from_user.id, *args, **kwargs)``
         """
         return self.bot.getUserProfilePhotos(self.id, *args, **kwargs)
+
+    @staticmethod
+    def de_list(data, bot):
```

Comment:
Hmm, so you're saying we shouldn't both have a de_list and a de_json, but that de_json should do both?